online_store/gt_sport_v2.html

```diff
@@ -0,0 +1,119 @@
+  <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sport.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="sportTitle1">goodTeens - Sports</h2> <p>
+<h2 class="sportTitle2">Online Store</h2>
+<p class="a"></p>
+<div class="sportWrapper">
+  <div class="card" id="product"> 
+    <div id="productDescription"></div>
+    <div id="productImg"></div>
+  </div>
+  <input type="button" id="addToSC" value="Add to Shopping Cart"/>
+</div>
+
+
+<script>
+const sportProducts = [
+  {
+    imgSrc: "skateboard.jpg",
+    name: "Skateboard",
+    brand: "Hudora",
+    level: "beginner",
+    price: 50,
+    seller: "Hudora",
+    other: {
+      helmetNeeded: true,
+      longevity: "mid-term",
+      teamSpirit: false,
+    }
+  },
+  {
+    imgSrc: "standuppaddle.jpg",
+    name: "Stand-up paddle",
+    brand: "Oceana",
+    level: "advanced",
+    price: 300,
+    seller: "Oceana",
+    other: {
+      helmetNeeded: false,
+      longevity: "long-term",
+      teamSpirit: true,
+    }
+  },
+  {
+    imgSrc: "volleyball.jpg",
+    name: "Volleyball ball",
+    brand: "Mikasa",
+    level: "beginner",
+    price: 26,
+    seller: "Mikasa",
+    other: {
+      helmetNeeded: false,
+      longevity: "short-term",
+      teamSpirit: true,
+    }
+    },
+  {
+    imgSrc: "xbadmintonset.jpg",
+    name: "Badminton set",
+    brand: "Talbot Torro",
+    level: "intermediate",
+    price: 28,
+    seller: "Talbot Torro",
+    other: {
+      helmetNeeded: false,
+      longevity: "mid-term",
+      teamSpirit: true,
+    }
+    },
+  {
+    imgSrc: "skigoggles.jpg",
+    name: "Ski Goggles",
+    brand: "Lemego",
+    level: "intermediate",
+    price: 28,
+    seller: "Lemego",
+    other: {
+      helmetNeeded: true,
+      longevity: "mid-term",
+      teamSpirit: true,
+    }
+    },
+];
+
+function productDescriptionDisplay() {
+  for (let i=0; i < sportProducts.length; i++) {
```

Comment:
this for loop is never closed. you need to add an `}` somewhere to close it. Correct indention makes this bug easier to spot

Reply:
Yep, found it, auto format document helped! Fixed! Thx